pages/common/curl.md

```diff
@@ -3,34 +3,34 @@
 > Transfers data from or to a server.
 > Supports most protocols including HTTP, FTP, POP.
 
-- Download a URL to a file:
+- Download the contents of an URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
 
-- Send form-encoded data:
+- Download a file saving it under the same name as indicated in the URL:
```

Comment:
Here the previous description ("Download a file saving the output under the same filename as indicated in the URL") would be more appropriate, IMO, to call attention to the `-O` option, which is different from `-o`.

Maybe we could shorten it a bit to "Download a file saving the output under the filename indicated by the URL", and we might consider writing [O]utput, to match the next examples.

Thoughts?